Lib/dataclasses.py

```diff
@@ -9,6 +9,7 @@
 import abc
 import _thread
 from types import FunctionType, GenericAlias
+from typing import ForwardRef
```

Comment:
I believe that up till now, there's been an attempt to avoid importing ``typing`` in ``dataclasses``, if at all possible, as it's quite an expensive import. It might be better to import this lazily in the ``_field_init`` function (i.e., only if we're sure we really need it)?